src/main/java/com/charlesgadeken/entwined/EntwinedTriggers.java

```diff
@@ -0,0 +1,390 @@
+        registerEffectControlParameter(colorStrobeTextureEffect.amount, "3707000050a946", 0, 1, 1);
+        registerEffectControlParameter(fadeTextureEffect.amount, "3707000050a967", 0, 1, 1);
+        registerEffectControlParameter(acidTripTextureEffect.amount, "3707000050a953", 0, 1, 1);
+        registerEffectControlParameter(candyCloudTextureEffect.amount, "3707000050a92d", 0, 1, 1);
+        registerEffectControlParameter(staticEffect.amount, "3707000050a8b3", 0, .3, 1);
+        registerEffectControlParameter(candyTextureEffect.amount, "3707000050aafc", 0, 1, 5);
+
+        effectKnobParameters =
+                new LXListenableNormalizedParameter[] {
+                    colorEffect.hueShift,
+                    colorEffect.mono,
+                    colorEffect.desaturation,
+                    colorEffect.sharp,
+                    blurEffect.level,
+                    speedEffect.speed,
+                    spinEffect.spin,
+                    candyCloudTextureEffect.amount
+                };
+    }
+
+    public void registerPattern(EntwinedBasePattern pattern, String nfcSerialNumber) {
+        registerPattern(pattern, nfcSerialNumber, 2);
+    }
+
+    public void registerPattern(
+            EntwinedTriggerablePattern pattern, String nfcSerialNumber, int apc40DrumpadRow) {
+        registerVisual(pattern, nfcSerialNumber, apc40DrumpadRow, VisualType.Pattern);
+    }
+
+    void registerVisual(
+            EntwinedTriggerablePattern pattern,
+            String nfcSerialNumber,
+            int apc40DrumpadRow,
+            VisualType visualType) {
+
+        LXBlend t = new DissolveBlend(lx);
+
+        // NOTE(meawoppl) @Slee same question below.  re `.setDuration(dissolveTime);`
+        pattern.setTransition(t);
+
+        Triggerable triggerable = configurePatternAsTriggerable(pattern);
+        BooleanParameter toggle = null;
+        if (apc40Drumpad != null) {
+            toggle =
+                    apc40DrumpadTriggerablesLists[apc40DrumpadRow].size() < 9
+                            ? nfcToggles[apc40DrumpadRow][
+                                    apc40DrumpadTriggerablesLists[apc40DrumpadRow].size()]
+                            : null;
+            apc40DrumpadTriggerablesLists[apc40DrumpadRow].add(triggerable);
+        }
+        if (nfcEngine != null) {
+            nfcEngine.registerTriggerable(nfcSerialNumber, triggerable, visualType, toggle);
+        }
+    }
+
+    Triggerable configurePatternAsTriggerable(EntwinedTriggerablePattern pattern) {
+        LXChannel channel = lx.engine.mixer.addChannel(new EntwinedBasePattern[] {pattern});
+        setupChannel(channel, false);
+
+        pattern.onTriggerableModeEnabled();
+        return pattern.getTriggerable();
+    }
+
+    void setupChannel(final LXChannel channel, boolean noOpWhenNotRunning) {
+        // @Slee, honestly no idea what the intention is here...
```

Comment:
If you want to add a custom transition type to the list of available types in the dropdown, you can do that in your initialize() callback via:

lx.registry.addChannelBlend()
http://lx.studio/api/heronarts/lx/LXRegistry.html#addChannelBlend-java.lang.Class-